game/src/core/level/elements/ILevel.java

```diff
@@ -482,7 +502,10 @@ default Point randomTilePoint() {
    * @return The position of a randomly selected tile of the specified type in the level as a {@link
    *     Point}.
    */
-  default Point randomTilePoint(final LevelElement elementType) {
-    return randomTile(elementType).position();
+  default Optional<Point> randomTilePoint(final LevelElement elementType) {
+    Optional<Tile> t = randomTile(elementType);
+    if (t.isPresent()) {
+      return Optional.of(t.get().position());
+    } else return Optional.empty();
```

Comment:
```suggestion
    return Optional.of(t.get().position()).orElse(Optional.empty());
```

Reply:
DANACH hab ich gesucht. Ich wusste es geht mehr sexy ich kam einfach nur nicht drauf.

Reply:
unsere vorstellung von "sexy" scheinen sich ... zu unterscheiden 🤣

aber ja, ich glaube, schöner wird es in java nicht.

Reply:
tatsächlich wird es so leider nicht funktionieren 🤔

das Optional.of() knallt bei null innen drin, und das orElse ist für das auspacken (und nicht einpacken) gedacht. 

aber so ähnlich sollte es gehen. vorne brauchst du die variante von Optional.of, die auch null-werte erlaubt. statt dem get nimmst du die variante, die zwei funktionen erlaubt. 

noch besser wäre aber, statt randomTile() eine verwandte funktion randomPosition() aufzurufen - das ist es ja, was du suchst. damit kannst du dir den anschließenden zauber mit aus- und umpacken komplett ersparen und das ergebnis direkt zurückliefern.

Reply:
@AMatutat Die korrekte Formulierung sollte `return randomTile(elementType).map(tile -> tile.position());` sein.

Reply:
siehe https://github.com/Dungeon-CampusMinden/Dungeon/pull/1624#pullrequestreview-2245871965 ... closing here.